style.css

```diff
@@ -0,0 +1,324 @@
+/* 
+Theme Name: Kety
+Text Domain: I'm Kety
+Version: 5.2
+Description: Custom Theme No.432
+Author: Katarina
+*/
+
+.h1,
```

Comment:
nemoj korisiti seleketore html-a za stiliziranje stvari kao što su h,p, img, itd. Koristi uvijek classe